VersionControlUI/src/DiffManager.h

```diff
@@ -141,12 +149,20 @@ class VERSIONCONTROLUI_API DiffManager
 
 		bool processNameChange(Model::Node* oldNode, Model::Node* newNode, const DiffSetup& diffSetup);
 
-		void showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup);
+		void showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup,
+												 VersionControlUI::DiffComparisonPair* diffComparisonPair);
+
+		bool satisfiesTargetNodeIdConstraint(Model::Node* oldNode, Model::Node* newNode,
```

Comment:
satisfiesTargetNodeIdConstraint checks whether the nodes are the ones we are interested in (same id as targetNodeId_) or targetNodeId_ is an ancestor of them.

The other method I explained in the other comment.